### ISSUE TITLE
Add alternative checksum type

### ISSUE THREAD

**Issue by alliomeria** [esmero/strawberryfield#233]
# What is needed?

Currently checksums are generated via MD5, which will remain the primary type. As a configuration option, add an alternative, supplementary checksum type for SHA256.

**Comment by DiegoPino** [comment]
@alliomeria Makes sense. We should provide both. Wonder if we would need a "Post process existing ones?" in case some chooses to do so?

**Comment by DiegoPino** [comment]
(HOPE NOT!). MD5 needs to be the main one, AWS S3 and many other internal parts of our stack use the good enough and fast MD5

**Comment by alliomeria** [comment]
Ah ok @DiegoPino --I updated the issue note above for clarification. :)
